### PR TITLE
tighten secondary build vendors

### PR DESCRIPTION
**Pull request by deads2k**
Move the builder related sub-vendor folder down to builders, not build.

### CODE REVIEW

**Comment by deads2k** [comment]
@soltysh @mfojtik Make sure that whatever you've done in the rebase still allows this.

**Comment by liggitt** [comment]
rebase has a bump of `pkg/build/vendor/github.com/Sirupsen/logrus`...

**Comment by deads2k** [comment]
> rebase has a bump of pkg/build/vendor/github.com/Sirupsen/logrus...

So I heard, we should merge this.

/test all

**Comment by deads2k** [comment]
@bparees FYI.  I expect you'd be unopinionated.

File: pkg/build/builder/util/dockerfile/dockerfile_test.go [hunk 1]
```diff
@@ -274,53 +274,6 @@ ENV PATH=/bin
 	}
 }
 
-// TestLastBaseImage tests calling LastBaseImage with multiple valid
-// combinations of input.
-func TestLastBaseImage(t *testing.T) {
```
Comment (bparees) [inline pkg/build/builder/util/dockerfile/dockerfile_test.go]: unopinionated except for the tests being deleted, what's the story there?

Comment (deads2k) [reply pkg/build/builder/util/dockerfile/dockerfile_test.go]: > unopinionated except for the tests being deleted, what's the story there?

The functions they were testing no longer exist.  Dead code.

Comment (bparees) [reply pkg/build/builder/util/dockerfile/dockerfile_test.go]: i guess that's a good reason.

**Comment by liggitt** [comment]
LGTM

**Pull request merged**